src/io/writer.js

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash'
+import stringify from 'csv-stringify'
+import Promise from 'bluebird'
+import iconv from 'iconv-lite'
+import Excel from 'exceljs'
+
+const fs = Promise.promisifyAll(require('fs'))
+
+const debugLog = _.noop
+// const debugLog = console.log
+
+export default class Writer {
+  constructor (options) {
+    this.options = options
```

Comment:
There is just one default option `defaultEncoding` which is used also somewhere else (line 43) so I store it in a variable instead..